endpoints/get.go

```diff
@@ -38,9 +46,16 @@ func NewGetHandler(storage backends.Backend, metrics *metrics.Metrics, allowCust
 
 func (e *GetHandler) handle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	e.metrics.RecordGetTotal()
+
+	if utils.RandomPick(e.cfg.refererLogRate) == true {
+		if refererHeaderValue := r.Header.Get(utils.REFERER_HEADER_KEY); refererHeaderValue != "" {
```

Comment:
Can we use the helper method here (and for the PUT handler) instead?

```
if referer := r.Referer(); referer != "" {
```

Reply:
Wow. I didn't know about this function. Looks a lot better. Modified.

Reply:
I expected there to a constant for the referer header name in the Go standard packages somewhere and found this instead.